### PR TITLE
A Spring starter for oracle nosql

### PR DESCRIPTION
**Pull request by cezarfx**
A Spring starter for Oracle NoSQL Spring SDK.
Autoconfiguration for Oracle NoSQL Spring SDK: configure and provide a NosqlDBConfig bean in the same style we do for other OCI services.

### CODE REVIEW

**Review by anders-swanson — DISMISSED** [review]

File: spring-cloud-oci/spring-cloud-oci-autoconfigure/src/main/java/com/oracle/cloud/spring/nosql/NoSQLAutoConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/*
+ ** Copyright (c) 2023, Oracle and/or its affiliates.
```
Comment (markxnelson) [inline spring-cloud-oci/spring-cloud-oci-autoconfigure/src/main/java/com/oracle/cloud/spring/nosql/NoSQLAutoConfiguration.java:2]: incorrect year in notice

Comment (markxnelson) [reply spring-cloud-oci/spring-cloud-oci-autoconfigure/src/main/java/com/oracle/cloud/spring/nosql/NoSQLAutoConfiguration.java:2]: applies to other files too, please check all

**Review by markxnelson — COMMENTED** [review]
please check years in notices are correct - should be first year, last year

**Review by anders-swanson — APPROVED** [review]
LGTM

**Pull request merged**